handbook/customers/sales.rituals.yml

```diff
@@ -0,0 +1,20 @@
+# https://github.com/fleetdm/fleet/pull/13084
+
+
+
+-
+  task: "Prioritize for next sprint" # Title that will actually show in rituals table
+  startedOn: "2023-09-04" # Needs to align with frequency e.g. if frequency is every thrid Thursday startedOn === any third thursday
+  frequency: "Triweekly" # must be supported by
+  description: "Drag next sprint's priorities to the top of the 'Not yet' column." # example of a longer thing: description: "[Prioritizing next sprint](https://fleetdm.com/handbook/company/communication)"
+  moreInfoUrl: "https://fleetdm.com/handbook/company/why-this-way#why-make-work-visible" #URL used to highlight "description:" test in table
+  dri: "alexmitchelliii" # DRI for ritual (assignee if autoIssue) (TODO display GitHub proflie pic instead of name or title)
+  autoIssue:   # Enables automation of GitHub issues
+    labels: [ "#g-sales" ] # label to be applied to issue
+-
+  task: "Saley thing"
```

Comment:
Write in imperative mood, this example does not match the handbook page framework.